h2o-py/h2o/frame.py

```diff
@@ -4643,26 +4643,26 @@ def stratified_split(self, test_frac=0.2, seed=-1):
         """
         return H2OFrame._expr(expr=ExprNode('h2o.random_stratified_split', self, test_frac, seed))
 
-    def match(self, table, nomatch=0):
+    def match(self, table, nomatch=float("nan"), start_index=1):
         """
-        Make a vector of the positions of (first) matches of its first argument in its second.
+        Makes a vector where an index of value from the table is returned if the value matches; returns ``nomatch`` value otherwise. 
 
-        Only applicable to single-column categorical/string frames.
-
-        :param List table: the list of items to match against
-        :param int nomatch: value that should be returned when there is no match.
-        :returns: a new H2OFrame containing for each cell from the source frame the index where
-            the pattern ``table`` first occurs within that cell.
+        :param List table: The list of items to match against.
+        :param int nomatch: Value that should be returned when there is no match. Numeric value or ``nan``. 
+        :param int start_index: Index from which this starts the indexing of the table list, numeric value >=0 (default is ``1``).
+        :returns: A new H2OFrame containing a vector where the index of value from the table is returned if the value matches; returns 
+            ``nomatch`` value otherwise. 
 
         :examples:
 
         >>> iris = h2o.import_file("http://h2o-public-test-data.s3.amazonaws.com/smalldata/iris/iris.csv")
-        >>> matchFrame = iris["C5"].match(['Iris-versicolor'])
-        >>> matchFrame
-        >>> matchFrame = iris["C5"].match(['Iris-setosa'])
-        >>> matchFrame
+        >>> match_col = iris["C5"].match(['Iris-setosa', 'Iris-versicolor'])
+        >>> match_col.names = ['match']
+        >>> iris_match = iris.cbind(match_col)
+        >>> iris_split = iris_match.split_frame(ratios=[0.05], seed=1)[0]
+        >>> print(iris_split)
         """
```

Comment:
is it possible to update this example to show the result as mentioned in previous comment https://github.com/h2oai/h2o-3/pull/15896#discussion_r1395969684?
It clarifies a lot what the function is doing, esp. for online doc.

same with R